transforms/remove-binding/index.js

```diff
@@ -0,0 +1,51 @@
+const { getParser } = require('codemod-cli').jscodeshift;
+
+module.exports = function transformer(file, api) {
+  const j = getParser(api);
+
+  const root = j(file.source);
+
+  const node = root.find(j.ObjectProperty, {
+    key: {
+      type: 'Identifier',
+    },
+  });
+
+  let filteredNodes = node.filter((path) => {
```

Comment:
💅🏽 just a naming suggestion, feel free to resolve this comment.

```suggestion
  let nodesWithBindings = node.filter((path) => {
```